### PR TITLE
Change "Latest updates" heading on homepage to "Recent updates"

### PR DESCRIPTION
**Pull request by Scotchester**
Closes https://GHE/CFGOV/platform/issues/3384

## Testing

1. Pull branch
1. Load http://localhost:8000/
1. See "Recent updates" instead of "Latest updates"

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by anselmbradford** [comment]
Should other mentions of this be aligned, https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/home_page.py#L30 and maybe this one https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L703

**Comment by Scotchester** [comment]
> Should other mentions of this be aligned, https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/home_page.py#L30 and maybe this one https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L703

Good question. The flag can be deleted, as it's no longer checked. I will add that to this PR.

The label on that field... meh. It doesn't seem worth creating a migration for that. I doubt it will cause any confusion.

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -700,9 +700,6 @@
     # Test financial well-being hub pages on Beta
     'FINANCIAL_WELLBEING_HUB': [('environment is', 'beta')],
 
-    # Home Page automatic Latest Updates
```
Comment (cfarm) [inline cfgov/cfgov/settings/base.py]: Is this a feature flag that is no longer needed?

Comment (Scotchester) [reply cfgov/cfgov/settings/base.py]: Correct.

**Comment by Scotchester** [comment]
gg codecov 🙄

**Pull request merged**